dotnet/samples/KernelSyntaxExamples/Example59_OpenAIFunctionCalling.cs

```diff
@@ -90,7 +90,7 @@ private static async Task CompleteChatWithFunctionsAsync(string ask, ChatHistory
         OpenAIFunctionResponse? functionResponse = chatResult.GetOpenAIFunctionResponse();
         if (functionResponse is not null)
         {
-            // If the function returned by OpenAI is an SKFunction registered with the kernel,
+            // If the function returned by OpenAI is an SKFunctionFactory registered with the kernel,
```

Comment:
If we're putting these on SKFunctionFactory, should we also move SKPlugin.FromXx to be SKPluginFactory.CreateFromXx?